packages/components/pie-chip/src/defs.ts

```diff
@@ -33,16 +36,30 @@ export interface ChipProps {
      * Can be only used if `isSelected` is set to true
      */
     isDismissible?: boolean;
+
+    /**
+     * Sets the functional type of the chip. Currently, only `checkbox` is supported.
```

Comment:
should we mention can be button (default) or checkbox?

Reply:
aw damn, I missed this - will update ty